es/api/DocumentsApi.js

```diff
@@ -85,6 +85,23 @@ module.exports = class DocumentsApi {
     return data;
   }
 
+  /**
+   * @param  {string} type The type of document to get
+   * @param  {string} id
+   * @return {Promise}
+   */
+  async get({ type, id }) {
```

Comment:
Holded didn't implement it when I first wrote the client but it's working now! 👍